### ISSUE TITLE
Custom Enchants

### ISSUE THREAD

**Issue by Duranduran05orDuran** [drewhoener/JWSuggestions#100]
There should some armor enchants like this:
Diamond: Add 500 Durabillaty to any armor BUT diamond Cannot be Repaired 1/85 chance
Body Guard: Like protechion but is stronger than Protechion (Less damage taken with Hearts) 1/55 chance
Glasses: On helments it gives Night Vision to the player Cannot Be repaired Breaks faster than normal 1/55 chance
Bird: On Boots it allows you to fly for 30 Seconds every 1 min 1/100 chance
Richer: Muptiplys coins amount per mob kill by 2 (Ex: Creeper is 3 but with richer its 6 3x6) 1/120 chance on swords

**Comment by TemperrroniPizza** [comment]
I dont think that would be added. It would be way too op, as well as the coin booster
